demo/Generic nonlinear delta node.ipynb

```diff
@@ -0,0 +1,193 @@
+{
```

Comment:
<p>Do we need this notebook? It seems to me a bit like it just explains some of the charateristics of <code>ReactiveMP</code>  (like dispatching on the <code>meta</code> object ) and not necessarily of the <code>DeltaFn</code>  node.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/biaslab/ReactiveMP.jl/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>